### ISSUE TITLE
Can not build with kernel 6.1 and 6.8.7

### ISSUE THREAD

**Issue by pbk20191** [YoSTEALTH/Shakti#1]
I can not install it from source. In either 6.1 Kernel and 6.8 Kernel of WSL2

[shakti_error.log](https://github.com/YoSTEALTH/Shakti/files/15066039/shakti_error.log)

**Comment by YoSTEALTH** [comment]
Yes, I just updated the code and I am having that issue as well. It will take time to debug, not really sure whats causing it atm. Currently there isn't much feature available for project Shakti as well, Its in very early starting phase of development. I want to get the core stabilized before adding features.

**Comment by YoSTEALTH** [comment]
@pbk20191 If you `python3 -m pip install --upgrade liburing` first and then `python3 -m pip install --upgrade git+https://github.com/YoSTEALTH/Shakti` that should work!

**Comment by YoSTEALTH** [comment]
Good news is you should be able to install the library now using `python3 -m pip install --upgrade git+https://github.com/YoSTEALTH/Shakti`.

Bad news is you will most likely run into Segfault issues. This library is in beginning stage! so ya...
